php_sql/electives.php

```diff
@@ -0,0 +1,78 @@
+<?php
+define("MAX_TITLE_LENGTH", 150, true);
+define("MAX_TEACHER_LENGTH", 200, true);
+define("MIN_DESCRIPTION_LENGTH", 10, true);
+define("MIN_CREDITS", 0, true);
+define("MAX_CREDITS", 20, true);
+
+$host = "localhost";
+$username = "root";
+$pass = "";
+$dbname = "test";
+
+function validateTitle($title, &$errors)
+{
+  if (!$title) {
+    $errors['title'] = "Името е задължително поле.";
+    echo $errors['title'];

```

Comment:
You have code duplication on lines **17 and 20**. This happens in every validation function.